### DISCUSSION TITLE
Annoyance: Gist "Tab Size" resets to 8 no matter what you do.

### DISCUSSION THREAD

**Discussion by gnat** [community/community#60839] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When using tabs on https://gist.github.com/ the tab size erroneously resets to 8 no matter what you set it to.

This experience sucks because tab size 8 looks ridiculous.

**Comment by kearfy** [comment]
Bump? I have the same issue. I set the option to tabs, with 4 spaces. result is tabs with 8 spaces. Also cannot change it back to spaces...

**Comment by gnat** [comment]
Yup. @kearfy Be sure to vote this up so this gets triaged. Thanks.

**Comment by sybrew** [comment]
This has been an issue for many, many years. 😟
